dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/StartupWarningMessagesTest.java

```diff
@@ -0,0 +1,73 @@
+package io.openliberty.microprofile.openapi20.fat.deployments;
+
+import static com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions.DISABLE_VALIDATION;
+import static com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions.SERVER_ONLY;
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+
+import java.util.Collections;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.microprofile.openapi20.fat.FATSuite;
+import io.openliberty.microprofile.openapi20.fat.deployments.test1.DeploymentTestApp;
+import io.openliberty.microprofile.openapi20.fat.deployments.test1.DeploymentTestResource;
+
+@RunWith(FATRunner.class)
+public class StartupWarningMessagesTest {
+    private static final String SERVER_NAME = "OpenAPIWarningMessageTestServer";
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = FATSuite.repeatDefault(SERVER_NAME);
+
+    @Test
+    public void tesWarningWhenMatchesOldNamingScheme() throws Exception {
+
+        EnterpriseArchive ear = null;
+        WebArchive war1 = ShrinkWrap.create(WebArchive.class, "test1.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource.class);
+
+        WebArchive war2 = ShrinkWrap.create(WebArchive.class, "test2.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource.class);
+
+        WebArchive war3 = ShrinkWrap.create(WebArchive.class, "test3.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource.class)
+                                    .addAsWebInfResource(DeploymentTestApp.class.getPackage(), "web.xml", "web.xml");
+
+        ear = ShrinkWrap.create(EnterpriseArchive.class, "testEar.ear")
+                        .addAsModules(war1, war2, war3);
+        ShrinkHelper.exportAppToServer(server, ear, SERVER_ONLY, DISABLE_VALIDATION);
+
+        server.setAdditionalSystemProperties(
+                                             Collections.singletonMap("mp_openapi_extensions_liberty_merged_include", "testEar"));
+        server.startServer();
+
+        //Test that a warning message saying we cant find the app but could under the pre OpenAPI 4.0 rules 
+        assertThat(
+                   server.findStringsInLogs(
+                                            "CWWKO1680W"),
+                   hasSize(1));
```

Comment:
The warning message is wrong, the rules differ depending on whether the config came from server.xml or MP Config, not whether we're using `mpOpenAPI-4.0` or an earlier feature.

We need to ensure the message has been substituted correctly as the name of the message also includes `CWWKO1680W`. Since this message has placeholders, we should include the expected values for those placeholders in the regex.

I'm not sure if this message is guaranteed to have been written by the time that `server.startServer()` returns. We should use `server.waitForStringInLogs` here unless we're certain that waiting is never necessary.